localstack-core/localstack/services/stepfunctions/provider.py

```diff
@@ -789,6 +790,10 @@ def start_execution(
             state_machine_arn_parts[1] if len(state_machine_arn_parts) == 2 else None
         )
 
+        # Count metrics about the execution type.
+        is_mock_test_case: bool = mock_test_case_name is not None
+        UsageMetrics.execution_type_counter.labels(is_mock_test_case=is_mock_test_case).increment()
```

Comment:
question: Are we missing other entry points, such as `start_sync_execution`? Are they worth tracking?